tools/spelling/spelling_dictionary.txt

```diff
@@ -269,6 +269,7 @@ RXQ
 Rebalance
 Runn
 SA
+SEGV
```

Comment:
For at least some of the words in this file, consider putting them in backticks in comments; that will make the spellchecker ignore them.

Reply:
I removed some PTAL